lib/web_ui/dev/README.md

```diff
@@ -23,6 +23,14 @@ felt build --watch
 
 If you don't want to add `felt` to your path, you can still invoke it using a relative path like `./web_ui/dev/felt <command>`
 
+## For googlers: accelerate your builds with goma
+If you are a Google employee you can use an internal instance of [goma](https://goto.google.com/goma) to speed up your builds.
+
+Once goma is installed and ready, you can take advantage of it in your builds:
+```
+felt build [-w] -j 100
+```
```

Comment:
Sorry for derailing this again, but now that we've figured out that `-j` is available to everyone, how about we make this useful for everyone? Strawman:

## Speeding up your builds

You can speed up your builds by using more CPU cores. Pass `-j` to specify the desired level of parallelism, like so:

```
felt build [-w] -j 100
```

If you are a Google employee you can use an internal instance of [Goma](https://goto.google.com/goma) to parallelize your builds. Because Goma compiles code on remote servers, this option is effective even on low-powered laptops.

Reply:
Thanks for writing this up 😃